src/components/forms/QuestionCreationForm.js

```diff
@@ -0,0 +1,195 @@
+// @flow
+
+import React from 'react'
+import _get from 'lodash/get'
+import isEmpty from 'validator/lib/isEmpty'
+import { connect } from 'react-redux'
+import { Field, reduxForm } from 'redux-form'
+import { FormattedMessage } from 'react-intl'
+
+import TitleInput from '../questions/creation/TitleInput'
+import TagInput from '../questions/creation/TagInput'
+import TypeChooser from '../questionTypes/TypeChooser'
+import stylesTagsInput from './styles-tagsinput'
+import {
+  SCCreationOptions,
+  SCCreationPreview,
+  SCCreationContent,
+} from '../../components/questionTypes/SC'
+
+import type { OptionType, TagType } from '../../types'
+
+// form validation
+const validate = ({ content, options, tags, title }) => {
+  const errors = {}
+
+  if (!title || isEmpty(title)) {
+    errors.title = 'form.createQuestion.title.empty'
+  }
+
+  if (!content || isEmpty(content)) {
+    errors.content = 'form.createQuestion.content.empty'
+  }
+
+  if (!options || options.length === 0) {
+    errors.options = 'form.createQuestion.options.empty'
+  }
+
+  if (!tags || tags.length === 0) {
+    errors.tags = 'form.createQuestion.tags.empty'
+  }
+
+  return errors
+}
+
+type Props = {
+  content: string,
+  intl: $IntlShape,
+  invalid: boolean,
+  options: Array<OptionType>,
+  tags: Array<TagType>,
+  title: string,
+  handleSubmit: () => void,
+  onDiscard: () => void,
+}
+
+const defaultProps = {
+  content: '',
+  options: [],
+  tags: [],
+  title: '',
+}
+
+const QuestionCreationForm = ({
+  intl,
+  invalid,
+  content,
+  options,
+  tags,
+  title,
+  handleSubmit: onSubmit,
+  onDiscard,
+}: Props) => (
+  <form className="ui form" onSubmit={onSubmit}>
+    <div className="questionInput questionTitle">
+      <Field name="title" component={TitleInput} />
+    </div>
+
+    <div className="questionInput questionType">
+      <Field name="type" component={TypeChooser} intl={intl} />
+    </div>
+
+    <div className="questionInput questionTags">
+      <Field name="tags" component={TagInput} tags={tags} />
+    </div>
+
+    <div className="questionInput questionContent">
+      <Field name="content" component={SCCreationContent} />
+    </div>
+
+    <div className="questionInput questionOptions">
+      <Field name="options" component={SCCreationOptions} />
+    </div>
+
+    <div className="questionPreview">
+      <SCCreationPreview title={title} description={content} options={options} />
+    </div>
+
+    <button className="ui button discard" type="reset" onClick={onDiscard}>
```

Comment:
This is what you meant with using basic css

Reply:
Yep!